phy/cluster/manual/gui.py

```diff
@@ -212,15 +218,31 @@ def func():
                 _add_gui_shortcut(_make_func(which, group))
 
     def _create_wizard_panel(self, cluster_ids=None):
-        view = _create_web_view(self.session.wizard._repr_html_())
+        styles = '''
```

Comment:
hm - we really want to shy away from this kind of stuff. much nicer to open it from a file... :/ will let it pass for now since this was supposed to be my job anyway! ;)